workspace/service.py

```diff
@@ -1,31 +1,42 @@
+import be_utilities.service_exceptions as service_exceptions
 import workspace.dao as ws_dao
 
+import user.service as user_service
+
+from db_python_util.db_exceptions import EntityNotFoundException
+
 # TODO: need to fix type annotations here
 
 def create(name: str, owner: str) -> int:
-    """Create a new workspace."""
+    if user_service.get(owner) == None:
```

Comment:
I'm glad we're checking this now! I think get will raise an error if the use can't be found rather than return None though (so I think we just need to swap the if with a try)

Reply:
actually based on the other comment i think this could be replaced by the user service's get (this probably applies a few more places too)